internal/data/timezone.go

```diff
@@ -0,0 +1,30 @@
+package data
+
+import (
+	"os"
+	"strings"
+)
+
+type TimeZoneService struct {
+	// time zone directory might be different for different systems
+	// timeZoneDir string
+}
+
+// Shameless copy from stack overflow https://stackoverflow.com/questions/40120056/get-a-list-of-valid-time-zones-in-go
+func (tz TimeZoneService) LoadTimeZones(path string) []string {
+	var timeZones []string
+	zoneDir := "/usr/share/zoneinfo/"
+	files, _ := os.ReadDir(zoneDir + path)
+	for _, f := range files {
+		if f.Name() != strings.ToUpper(f.Name()[:1])+f.Name()[1:] {
```

Comment:
Can there be a comment on what exactly `strings.ToUpper(f.Name()[:1])+f.Name()[1:]` represent?

Reply:
Yeah, I think I'm going to rewrite this code. It's unnecessarily complex and depends on the OS being used. Do you think hard coding the list is an alright idea?

Reply:
I feel it is a bad idea plus this will not work for all OSes out there.

Reply:
I think hardcoding a list of timezones is a great idea!